tendrl/commons/objects/definition/master.yaml

```diff
@@ -1165,4 +1165,14 @@ namespace.tendrl:
       value: /clusters/{0}/nodes/{1}/alert_counters
       list: /clusters/{0}/nodes/{1}/alert_counters
       help: "Cluster Alert Counter"
+    Index:
+      enabled: True
+      attrs:
+        key:
```

Comment:
Here I saw key is an attribute for Index but in Index class attributes key is not used as a member variable, any reason for that?

Reply:
Is it not necessary to present as a member variable?